cristin-person/src/test/java/no/unit/nva/cristin/person/fetch/FetchCristinPersonHandlerTest.java

```diff
@@ -306,6 +311,22 @@ void shouldNotHaveEmploymentFieldInResponseWhenNotInUpstreamPayload() throws IOE
         assertThat(actual.getEmployments(), equalTo(null));
     }
 
+    @Test
+    void shouldReturnResponseContainingNviDataWhenPresentInUpstream() throws Exception {
```

Comment:
Always something new to learn from your tests....

Reply:
What did you learn?